eiffel-gerrit-plugin-script

```diff
@@ -100,18 +96,40 @@ function do_full_test {
   do_stop
 }
 
+function set_api_host {
+  container_name="$1"
+
+  unamestr=$(uname)
+  if [[ "$unamestr" == "Linux" ]]; then
+    CONTAINER_ID=$(docker ps | grep ${container_name} | awk '{print $1}')
+    API_HOST=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $CONTAINER_ID)
```

Comment:
I think this line was required in VirtualBox in Windows, due to localhost is not working as it nomally do on a normal host.
But I guess this works on normal Linux installation as well, both container ip and localhost as host address.